### DISCUSSION TITLE
Is it possible to yield RunRequests to a dynamic job name on a sensor, without defining the job name on the sensor decorator?

### DISCUSSION THREAD

**Discussion by biancarosa** [dagster-io/dagster#26764] [Q&A]
I currently have a sensor that I am trying to get it to work - it is the simplest sensor possible, using just the `@sensor()` decorator, with no arguments.

This sensor yields run requests based on a few parameters that come from an external source. However, Dagster is currently complaining that I need to define a job_name on the sensor definition.

Is what I'm trying to do - kickoff a job dynamically on a sensor - possible?

**Comment by garethbrickman** [comment]
I believe you can use the `jobs` parameter to specify a list of jobs and then in your sensor code you can selectively yield RunRequests for them.

**Reply by biancarosa** [reply]
So in case I can trigger `any` job in my repo, I would pass the full list of jobs in the repo to the sensor? Seems a bit of an overkill.

**Reply by garethbrickman** [reply]
Yeah unfortunately it seems we can't yet dynamically load all jobs the way we can with assets. It is requested here: https://github.com/dagster-io/dagster/issues/12359
